xml/gnome_custom.xml

```diff
@@ -133,17 +121,16 @@
    <itemizedlist>
     <listitem>
      <para>
-      <guimenu>Language</guimenu>. Choose a language from the list and confirm
-      with <guimenu>Done</guimenu>. To make the change take effect, you need to
-      restart the desktop afterwards.
+      <guimenu>Language</guimenu>. Choose a language from the list. To make the
+      change take effect, you will be prompted to restart the desktop session
+      afterwards.
      </para>
     </listitem>
     <listitem>
      <para>
       <guimenu>Formats</guimenu>: To change the formats for date, number,
-      currency and related options, choose a country from the list and confirm
-      with <guimenu>Done</guimenu>. To make the change take effect, you need to
-      restart the desktop afterwards.
+      currency and related options, choose a country from the list. To make the
+      change take effect, you need to restart the desktop afterwards.
```

Comment:
As per previous comment, we could say:

"To make the changes take effect, you must log out of your session and log back in."